docs/h264.md

```diff
@@ -218,9 +218,11 @@ The client-side JavaScript application uses the following control flow:
             // mediaStreamTrack creates a new object with a distinct, globally
```

Comment:
Github won't let me comment on the exact line (214), but can we change "for when the **video** stream arrives" to "when the media stream arrives"?

Reply:
I've updated this, and comments/prose throughout.